### PR TITLE
Gamedb: Cleanup/correct formatting and code comments v2.

### PR DESCRIPTION
**Pull request by RedDevilus**

### CODE REVIEW

**Comment by bositman** [comment]
I'm not sure how helpful it is to include translated game names of the PAL regions, someone wanting to change all game entries across all regions might have trouble this way...

**Comment by RedDevilus** [comment]
I changed it to one name, people can look at the region itself if they want to know what the title is on psxdatabase or pcsx2 wiki.

File: bin/GameIndex.dbf [hunk 1]
```diff
@@ -1438,15 +1438,15 @@ Name   = Monsters Inc.
 Region = PAL-DU
 ---------------------------------------------
 Serial = SCES-50598
-Name   = Monsterit Oy
+Name   = Monsters Inc.
```
Comment (pgert) [inline bin/GameIndex.dbf:1441]: Why? If you want clarity use "Monsterit Oy	// Monsters Inc.".

File: bin/GameIndex.dbf [hunk 2]
```diff
@@ -1458,16 +1458,16 @@ Name   = Monsters Inc.
...
 ---------------------------------------------
 Serial = SCES-50603
-Name   = Monstruos S.A - La Isla de los Sustos
+Name   = Monsters Inc.
 Region = PAL-S
 ---------------------------------------------
 Serial = SCES-50604
 Name   = Monsters Inc.
-Region = PAL-Unk
+Region = PAL-SWE
```
Comment (pgert) [inline bin/GameIndex.dbf:1466]: "PAL-SE", not "PAL-SWE".

File: bin/GameIndex.dbf [hunk 3]
```diff
@@ -36270,7 +36264,7 @@ Name   = Kessen 2
 Region = NTSC-U
 Compat = 5
 [patches = 53A803AF]
-	comment=COP2 flag instance patch by refraction
+	comment=COP2 flag instance patch by Refraction
```
Comment (pgert) [inline bin/GameIndex.dbf:36267]: Isn't it "refraction"?

Comment (refractionpcsx2) [reply bin/GameIndex.dbf:36267]: It is "refraction" I don't use a capital R

File: bin/GameIndex.dbf [hunk 4]
```diff
@@ -743,7 +743,7 @@ Name   = Namco x Capcom
 Region = NTSC-Unk
 ---------------------------------------------
 Serial = SCAJ-20131
-Name   = namCollection - Namco 50th Anniversary
+Name   = NamcoCollection - Namco 50th Anniversary
```
Comment (refractionpcsx2) [inline bin/GameIndex.dbf:746]: No, it's literally called namCollection, look at the box https://s.pacn.ws/1500/PA.28798.017.jpg?o2utd0

File: bin/GameIndex.dbf (hunk 1 above)
Comment (refractionpcsx2) [inline bin/GameIndex.dbf:1441]: Please keep the french name, it's not called "Monsters Inc." in France.

File: bin/GameIndex.dbf [hunk 5]
```diff
@@ -1438,15 +1438,15 @@ Name   = Monsters Inc.
...
+Name   = Monsters Inc.
 Region = PAL-FI
 ---------------------------------------------
 Serial = SCES-50599
 Name   = Monsters Inc.
 Region = PAL-F
 ---------------------------------------------
 Serial = SCES-50600
-Name   = Disney's Monster Inc. - Scare Island
+Name   = Monsters Inc.
```
Comment (refractionpcsx2) [inline bin/GameIndex.dbf:1449]: This is actually called Disney Pixar - Die Monster AG

File: bin/GameIndex.dbf [hunk 6]
```diff
@@ -1458,16 +1458,16 @@ Name   = Monsters Inc.
 Region = PAL-Unk
 ---------------------------------------------
 Serial = SCES-50603
-Name   = Monstruos S.A - La Isla de los Sustos
+Name   = Monsters Inc.
```
Comment (refractionpcsx2) [inline bin/GameIndex.dbf:1461]: This is called Disney Pixar - Monstruos S.A. - La Isla De Los Sustos

File: bin/GameIndex.dbf (hunk 2 above)
Comment (refractionpcsx2) [reply bin/GameIndex.dbf:1466]: Sweden is never written SWE

File: bin/GameIndex.dbf [hunk 7]
```diff
@@ -2741,7 +2741,7 @@ Name   = ATV Offroad Fury 4
 Region = PAL-M12
 ---------------------------------------------
 Serial = SCES-54748
-Name   = WipEout Pulse
+Name   = Wipeout Pulse
```
Comment (refractionpcsx2) [inline bin/GameIndex.dbf:2744]: WipEout was correct.

File: bin/GameIndex.dbf [hunk 8]
```diff
@@ -9730,7 +9730,7 @@ Compat = 4 // This game needs the EE cache to work which is only supported in "i
 	author=PSI
 	// The game allocates a VIF DMA buffer on the stack and sends it, returns from the function, then calls another function to wait for the transfer to end.
 	// The second function causes stack corruption as it allocates over the buffer, which is still transferring data.
-	// This works on real hardware due to the data cache - the transfer will remain safe in main memory, and the stack corruption only happens in cache.
+	// This works on real Hardware due to the data cache - the transfer will remain safe in main memory, and the stack corruption only happens in cache.
```
Comment (refractionpcsx2) [inline bin/GameIndex.dbf:9733]: "hardware" is not a name, it's a word in the middle of a sentence, it doesn't get capitalised.

File: bin/GameIndex.dbf [hunk 9]
```diff
@@ -37029,7 +37023,7 @@ Compat = 5 // This game needs the EE cache to work which is only supported in "i
...
```
Comment (refractionpcsx2) [inline bin/GameIndex.dbf:37026]: don't capitalise hardware here.

File: bin/GameIndex.dbf [hunk 10]
```diff
@@ -38154,7 +38148,7 @@ Compat = 4
 EETimingHack = 1
 [patches = B67F4F9E]
 	comment=Patch by Prafull
-	// Avoids hanging before molotov party.
+	// Avoids hanging before Molotov party.
```
Comment (refractionpcsx2) [inline bin/GameIndex.dbf:38151]: if Molatov Party is a quest name, shouldn't the P be capitalised too?

**Comment by bositman** [comment]
I seriously dunno why all of us are wasting time for capitalizing words in a db file. Surely there are much better things to use our time at for improving PCSX2? :P

File: bin/GameIndex.dbf [hunk 11]
```diff
@@ -1,5 +1,5 @@
 ---------------------------------------------
--- PCSX2 Game DataBase!
+-- PCSX2 Game Database!
```
Comment (lightningterror) [inline bin/GameIndex.dbf:2]: I prefer old.

Comment (refractionpcsx2) [reply bin/GameIndex.dbf:2]: Wiki says Database, I agree with that tbh.

**Comment by RedDevilus** [comment]
I did notice alot of Compat = 5  (Playable) is missing for games. Should i ignore those?

**Comment by lightningterror** [comment]
Ignore them.

File: bin/GameIndex.dbf (hunk 11 above)
Comment (RedDevilus) [reply bin/GameIndex.dbf:2]: Correct use is Database and it fits better is the reason i changed it.

File: bin/GameIndex.dbf [hunk 12]
```diff
@@ -9946,15 +9946,9 @@ Name   = Sega Bass Fishing Duel
...
-Name   = Fightbox
+Name   = FightBox
 Region = PAL-E
 vuClampMode = 0 // Fixes missing 3D
-[patches = E7EFA0BF]
-	comment= Gets around GS SIGNAL hang, game expects value other than C in SIGNAL but the PATH2 packet sends double 0xC
-	comment= The SIGNAL value is compared to the value at address 0x34CDF3, if this address contains 0x4 (OR'd with 0x8)
-	commend= the interrupt doesn't get acknoledged and cleared and so the PATH2 packet hangs
-	patch=1,EE,001f5a0c,word,30840008
-[/patches]
```
Comment (lightningterror) [inline bin/GameIndex.dbf:9957]: Why was this removed?

Comment (RedDevilus) [reply bin/GameIndex.dbf:9957]: This was already removed in the latest dev builds where ref fixed it, somehow github says it gets deleted again (which is wrong). It's the same as there. Not sure why its saying that about Fightbox

Comment (lightningterror) [reply bin/GameIndex.dbf:9957]: A bit odd, maybe rebase with latest master?

Comment (RedDevilus) [reply bin/GameIndex.dbf]: Huh, it was missing a commit, but it reverted that commit for some reason. Anyway it seem to have being fixed.
